ex2/ex2.c

```diff
@@ -1,14 +1,33 @@
-// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory 
-// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor 
+// Write a program that opens the text.txt  file (with the `fopen()` system call) located in this directory
+// and then calls `fork()` to create a new process. Can both the child and parent access the file descriptor
 // returned by `fopen()`? What happens when they are written to the file concurrently?
+//they're both able to run on a single fopen. Interestingly, throwing an fclose at the end of either the if or else statement doesn't
+//seem to do anything
 
 #include <stdio.h>
 #include <unistd.h>
 #include <stdlib.h>
 
 int main(void)
 {
-    // Your code here 
-    
-    return 0;
+    FILE *fp;
+
+    fp = fopen("text.txt", "w+");
+    int count = fork();
+    if (count == 0)
+    {
+        fprintf(fp, "%s %s %s %s", "I", "can", "go", "anywhere\n");
+        printf("child printed\n");
+        fclose(fp);
+        printf("child closed\n");
+    }
+    else
+    {
+        fprintf(fp, "%f %f %s %s", 8.03, 67.57, "do", "anything\n");
```

Comment:
This is interesting, I liked that you use floats here. It's good to showcase that not only strings can be printed/written to files.